test/functional/editor/jump_spec.lua

```diff
@@ -182,6 +182,109 @@ describe("jumpoptions=stack behaves like 'tagstack'", function()
   end)
 end)
 
+describe('buffer deletion', function()
+  local base_file = 'Xtest-functional-buffer-deletion'
+  local file1 = base_file..'1'
+  local file2 = base_file..'2'
+  local file3 = base_file..'3'
+  local base_content = 'text'
+  local content1 = base_content..'1'
+  local content2 = base_content..'2'
+  local content3 = base_content..'3'
+  local function delete_buf(file)
+    local bufnr = funcs.bufnr(file)
+    command('bw '..bufnr)
+  end
+
+  before_each(function()
+    clear()
+    feed(':clearjumps<cr>')
```

Comment:
is feed() intentional or can command be used? feed() is asynchronous and hides errors.

```suggestion
    command('clearjumps')
```